lib/users/query.js

```diff
@@ -104,6 +104,20 @@ class BaseQuery {
     return this.addSearch(field, searchStrs.join('|'));
   }
 
+  /**
+   * Adds the given search string to be applied over all
+   * searchable fields. The resulting query will return
```

Comment:
The existing convention here is to break the docstring into a one line summary and then an optional paragraph below of more detail. This is meaningful because some autodoc generation tools format the first line and the next paragraph differently.

Reply:
Updated. I thought that was a python only thing. My fault!